### PR TITLE
Added required permissions missing from documentation

### PR DESCRIPTION
**Pull request by marco-lancini**
Address Issue #212, by extending the README with additional permissions required by Cartography and not previously mentioned

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -416,3 +431,8 @@ There are many ways to allow Cartography to pull from more than one AWS account.
 
 		... etc ...
 		```
+
+
+#### Multiple GCP Project Setup
+
+In order for Cartography to be able to pull all assets from all GCP Projects within an Organization, the User/Service Account assigned to Cartography needs to be created at the **Organization** level.
```
Comment (achantavy) [inline README.md]: Do you have a reference on this/are there other official Google links that we provide here?

File: README.md [hunk 2]
```diff
@@ -381,12 +390,18 @@ There are many ways to allow Cartography to pull from more than one AWS account.
...
+				  "Effect": "Allow",
+				  "Action": "ec2:DescribeRegions",
+				  "Resource": "*"
+				}
 			  ]
 			}
 			```
 
-			This allows the Hub role to assume the `cartography-read-only` role on your Spoke accounts.
+			This allows the Hub role to assume the `cartography-read-only` role on your Spoke accounts and to fetch all the different regions used by the Spoke accounts.
```
Comment (achantavy) [inline README.md]: Ahh, good callout.

**Comment by achantavy** [comment]
Also it looks like CI is angry: I think you need to do `make test_lint` and then commit and push.

File: README.md (hunk 1 above)
Comment (marco-lancini) [reply README.md]: Good point, I've just added a reference

**Comment by achantavy** [comment]
Thanks for updating the docs!

**Pull request merged**